### ISSUE TITLE
No LICENSE / COPYING file

### ISSUE THREAD

**Issue by flihp** [01org/TPM2.0-TSS#67]
We've got a license header on the source files but there's no LICESE or COPYING file in the project root. This is a convention intended to make the license for the project obvious / easy to find.

**Comment by wcarthur1** [comment]
Well that’ll be interesting since there are about 3 different license headers.  Most have an Intel license, a few have a TCG license, and 1 or 2 have an Intel and Fraunhoffer license.

Will Arthur
Intel Corporation
Server Security Firmware
803-216-2101

**Comment by flihp** [comment]
@wcarthur1 : perfect example of why we need to make the project's license (plural) clear. Sounds like you may know where these different license are lurking. Can you plz provide pointers? The TCG one is probably in SAPI / TCTI headers, the Intel one is all over the place ... who's Fraunhoffer?

**Comment by wcarthur1** [comment]
Philip,

All the files that have a different copyright are in sysapi\include:
Basetypes.h, implementation.h, tpmb.h:  these have the TCG copyright
Tss2_common.h, tss2_tcti.h: these have the combined Intel/Fraunhofer copyright.

Fraunhofer is a German company—Andreas on the TSS WG works for them and contributed to the two files that have the Intel/Fraunhofer copyright.

Everything else is Intel copyright.

You may want to pass this through Intel legal.  I originally did this when we first open-sourced this code.

Will Arthur
Intel Corporation
Server Security Firmware
803-216-2101

**Comment by flihp** [comment]
I took a poke around and two things stand out. We need to keep from confusing "license" and "copyright". These are very different things (according to my reading of wikipedia).  So from my poking around it looks like there are a few different copyrights asserted on the sources but I've only been able to find one license that is basically the 2-clause BSD license.

Either way I think your advice to consult legal is probably the right thing to do. I'll report back.
